homeassistant/components/nissan_leaf/device_tracker.py

```diff
@@ -42,5 +39,3 @@ def see_vehicle():
                 icon=ICON_CAR)
 
     dispatcher_connect(hass, SIGNAL_UPDATE_LEAF, see_vehicle)
-
-    return True
```

Comment:
Here we should `return True`. So revert this.

Reply:
Fixed with commit https://github.com/home-assistant/home-assistant/pull/21145/commits/55647253d143fa6c4e306d73f2a55231f2131059